torch/csrc/jit/serialization/export.cpp

```diff
@@ -85,6 +85,7 @@ constexpr static std::array<int64_t, kMainOpsetVersion + 1>
         8, // opset 16
         8, // opset 17
         8, // opset 18
+        8, // opset 19
```

Comment:
```suggestion
        9, // opset 19
```
Not entirely sure, but I think opset 19 uses ir 9? Check this out https://github.com/onnx/onnx/blob/main/onnx/onnx.proto3#L106-L108